Source/Core/DiscIO/Blob.cpp

```diff
@@ -177,23 +178,24 @@ std::unique_ptr<IBlobReader> CreateBlobReader(const std::string& filename)
   if (cdio_is_cdrom(filename))
     return DriveReader::Create(filename);
 
-  if (!File::Exists(filename))
+  File::IOFile file(filename, "rb");
+  u32 magic;
```

Comment:
`magic` in this case indicates a [set of bytes](https://en.wikipedia.org/wiki/File_format#Magic_number) at the beginning of the file used to identify the file format.